app/packages/looker/src/elements/common/util.ts

```diff
@@ -41,7 +42,17 @@ export const dispatchTooltipEvent = <State extends BaseState>(
         ? overlays[0].getPointInfo(state)
         : null;
 
-    // @ts-ignore
+    if (detail?.label?.instance_config) {
+      document.dispatchEvent(
```

Comment:
I think it would be cleaner to implement a new EventTarget class and create a global singleton instance rather than add custom events to `document`

Spotlight is an `EventTarget`
* https://github.com/voxel51/fiftyone/blob/d9cf67d2f44a764f24daef1bfe0c369db16b2e84/app/packages/spotlight/src/index.ts#L47
* https://github.com/voxel51/fiftyone/blob/d9cf67d2f44a764f24daef1bfe0c369db16b2e84/app/packages/spotlight/src/index.ts#L82
* https://github.com/voxel51/fiftyone/blob/develop/app/packages/spotlight/src/events.ts